Explicitly recommend to forceMerge before freezing (#36376)
elastic/elasticsearch: docs/reference/frozen-indices.asciidoc

Since indices that are eligible for freezing are unlikely to change in the future, disk space can be optimized as described in <<tune-for-disk-usage>>.

> It's highly recommended to <<indices-forcemerge,`_forcemerge`>> you indices prior to freezing them to ensure the indices only have a single

Reviewer: ```suggestion
It's highly recommended to <<indices-forcemerge,`_forcemerge`>> your indices prior to freezing to ensure that each shard has only a single
```

> It's highly recommended to <<indices-forcemerge,`_forcemerge`>> you indices prior to freezing them to ensure the indices only have a single

Reviewer 2: ```suggestion
It's highly recommended to <<indices-forcemerge,`_forcemerge`>> your indices prior to freezing them to ensure the indices only have a single
```

> segment on disk. This will provide much better compression but also prevents the need for additional data-structures required for

Reviewer: ```suggestion
segment on disk. This will not only provide much better compression but also simplify the data structures needed to service aggregation requests.
```

Reviewer 2: You might also consider removing the future tense here.  For example: "This not only provides much better compression but also simplifies..."

Author: ++

> aggregation to be build on every request.

Reviewer: ```suggestion
```
